web/cobrands/fixmystreet.com/layout.scss

```diff
@@ -156,29 +156,69 @@ body.fullwidthpage {
 }
 
 body.frontpage {
+    $fms-logo-width: 200px;
+    $fms-logo-height: 60px;
+    $fms-logo-vertical-margin: 2em;
+    $mysociety-badge-width: 120px;
     #site-logo {
-        margin: 2em 0;
-        width: 300px;
-        height: 60px;
+        margin: $fms-logo-vertical-margin 0;
+        // width: 300px; Size without the mySociety badge
+        width: $fms-logo-width;
+        margin-left: $mysociety-badge-width;
+        position: relative;
+        height: $fms-logo-height;
         background: none;
 
         // Override the width="175" height="35" attributes on the svg element,
         // forcing the SVG contents to stretch to the new, larger, size.
         svg {
-            width: 300px;
-            height: 60px;
+            // width: 300px; Size without the mySociety badge
+            width: $fms-logo-width;
+            float: right;
+            height: $fms-logo-height;
             margin-top: 0; // no need for the vertical centering any more
         }
     }
 
+    #mysociety-badge {
+        top: calc((#{$fms-logo-height} + #{$fms-logo-vertical-margin}) * -1);
+        width: $mysociety-badge-width;
+        max-height: calc((#{$fms-logo-height} + #{$fms-logo-vertical-margin}) * 2);
+        svg {
+            width: $mysociety-badge-width;
```

Comment:
You set a `width` for the SVG, but not a `height`, meaning that keeps the original 266px height from the inline `height="266"` attribute. As a result, the hitbox for the SVG’s wrapper link is much taller than the SVG itself, which is a bit weird:

<img width="231" alt="Screenshot 2023-01-31 at 15 59 30" src="https://user-images.githubusercontent.com/739624/215814628-93809cc5-debe-4d98-bbf6-356d40d6fba6.png">

I’d set `height: auto` on the SVG. In fact, I’d probably leave all the sizing to `#mysociety-badge` instead, with something like this:

```scss
#mysociety-badge {
  width: $mysociety-badge-width;

  svg {
    width: 100%;
    height: auto;
  }
}
```